src/InfluxDB.LineProtocol/LineProtocolWriter.cs

```diff
@@ -0,0 +1,239 @@
+﻿using System;
+using System.Globalization;
+using System.IO;
+
+namespace InfluxDB.LineProtocol
+{
+    public class LineProtocolWriter
+    {
+        private static readonly DateTime UnixEpoch = new DateTime(1970, 1, 1, 0, 0, 0, DateTimeKind.Utc);
+
+        private readonly TextWriter textWriter;
```

Comment:
TextWriter.ToString() is not specifically implemented. Only StringWriter specifically implements it to return the StringBuilder content. Other TextWriter implementations could not return the content. You could change the field type to StringWriter instead to prevent this supposition.